home/pi/auto_run.sh

```diff
@@ -109,8 +108,7 @@ then
    # to picroft scripts, mycroft-core, etc.
```

Comment:
Just before this let's insert the following code:

```
   MARK1_ARDUINO_SCRIPT="/opt/mycroft/enclosure/upload.sh"
   if [ -f $MARK1_ARDUINO_SCRIPT ]
   then
       # This file slipped onto a release of Picroft accidentally.  It can
       # cause the CPU to rev a core to 100% while it attempts to update a
       # non-existant Arduino.
       sudo rm $MARK1_ARDUINO_SCRIPT
   fi
```